### ISSUE TITLE
anaconda spyder an error occurred while starting the kernel. unexpected EOF while parsing

### ISSUE THREAD

**Issue by honey09280** [spyder-ide/spyder#9886]
### Issue Report Checklist

I looked at the relevant issues in the past and followed the trouble shooting steps told in that issue. For example, I tried: 

conda update spyder ipykernel tornado pyzmq

conda install -c anaconda spyder ipykernel=4.8.2

But didn't fix the issue.

## Problem Description
Hi,

I recently installed Anaconda 2.7 on my Windows 64 bit Laptop. When I started/opened the spyder notebook from Anaconda, it says: 

an error occurred while starting the kernel. 

The error is:

Traceback (most recent call last):
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\spyder\plugins\ipythonconsole.py", line 1554, in create_kernel_manager_and_kernel_client
kernel_manager.start_kernel(stderr=stderr_handle)
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\jupyter_client\manager.py", line 254, in start_kernel
env.update(self.kernel_spec.env or {})
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\spyder\utils\ipython\kernelspec.py", line 113, in env
'startup/use_run_file'),
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\spyder\config\user.py", line 384, in get
value = ast.literal_eval(value)
File "C:\Users\HONEY\Anaconda2\lib\ast.py", line 49, in literal_eval
node_or_string = parse(node_or_string, mode='eval')
File "C:\Users\HONEY\Anaconda2\lib\ast.py", line 37, in parse
return compile(source, filename, mode, PyCF_ONLY_AST)
File "", line 0

^
SyntaxError: unexpected EOF while parsing

I haven't typed any code, I just opened spyder(3.3.2) from Anaconda and I see this in the console.

### What steps reproduce the problem?

1.  When I open the spyder, it gives error message in the console. Cannot execute any programs because of this issue,.
2. 
3. 

### What is the expected output? What do you see instead?

An error ocurred while starting the kernel
The error is:

Traceback (most recent call last):
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\spyder\plugins\ipythonconsole.py", line 1554, in create_kernel_manager_and_kernel_client
kernel_manager.start_kernel(stderr=stderr_handle)
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\jupyter_client\manager.py", line 254, in start_kernel
env.update(self.kernel_spec.env or {})
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\spyder\utils\ipython\kernelspec.py", line 113, in env
'startup/use_run_file'),
File "C:\Users\HONEY\Anaconda2\lib\site‑packages\spyder\config\user.py", line 384, in get
value = ast.literal_eval(value)
File "C:\Users\HONEY\Anaconda2\lib\ast.py", line 49, in literal_eval
node_or_string = parse(node_or_string, mode='eval')
File "C:\Users\HONEY\Anaconda2\lib\ast.py", line 37, in parse
return compile(source, filename, mode, PyCF_ONLY_AST)
File "", line 0

^
SyntaxError: unexpected EOF while parsing

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Versions

* Spyder version: 3.3.2
* Python version: 2.7.10
* Qt version: 
* PyQt version: 
* Operating System name/version: Windows 10 64 bit Laptop

### Dependencies

```
PASTE DEPENDENCIES HERE
```
alabaster                          0.7.12
anaconda-client                    1.7.2
anaconda-navigator                 1.9.7
anaconda-project                   0.8.2
asn1crypto                         0.24.0
astroid                            1.6.5
astropy                            2.0.9
atomicwrites                       1.3.0
attrs                              19.1.0
Babel                              2.6.0
backports-abc                      0.5
backports.functools-lru-cache      1.5
backports.os                       0.1.1
backports.shutil-get-terminal-size 1.0.0
backports.shutil-which             3.5.2
backports.ssl-match-hostname       3.7.0.1
beautifulsoup4                     4.7.1
bitarray                           0.8.3
bkcharts                           0.2
bleach                             3.1.0
bokeh                              1.0.4
boto                               2.49.0
Bottleneck                         1.2.1
cdecimal                           2.3
certifi                            2019.6.16
cffi                               1.12.2
chardet                            3.0.4
Click                              7.0
cloudpickle                        0.8.0
clyent                             1.2.2
colorama                           0.4.1
comtypes                           1.1.7
conda                              4.7.10
conda-build                        3.17.8
conda-package-handling             1.3.11
conda-verify                       3.1.1
configparser                       3.7.3
contextlib2                        0.5.5
cryptography                       2.4.2
cycler                             0.10.0
Cython                             0.29.6
cytoolz                            0.9.0.1
dask                               1.1.4
decorator                          4.4.0
defusedxml                         0.5.0
distributed                        1.26.0
docutils                           0.14
entrypoints                        0.3
enum34                             1.1.6
et-xmlfile                         1.0.1
fastcache                          1.0.2
filelock                           3.0.10
Flask                              1.0.2
funcsigs                           1.0.2
functools32                        3.2.3.post2
future                             0.17.1
futures                            3.2.0
gevent                             1.4.0
glob2                              0.6
greenlet                           0.4.15
grin                               1.2.1
h5py                               2.9.0
heapdict                           1.0.0
html5lib                           1.0.1
idna                               2.8
imageio                            2.5.0
imagesize                          1.1.0
importlib-metadata                 0.0.0
ipaddress                          1.0.22
ipykernel                          4.10.0
ipython                            5.8.0
ipython-genutils                   0.2.0
ipywidgets                         7.4.2
isort                              4.3.16
itsdangerous                       1.1.0
jdcal                              1.4
jedi                               0.13.3
Jinja2                             2.10
jsonschema                         3.0.1
jupyter                            1.0.0
jupyter-client                     5.2.4
jupyter-console                    5.2.0
jupyter-core                       4.4.0
jupyterlab                         0.33.11
jupyterlab-launcher                0.11.2
keyring                            18.0.0
kiwisolver                         1.0.1
lazy-object-proxy                  1.3.1
libarchive-c                       2.8
linecache2                         1.0.0
llvmlite                           0.28.0
locket                             0.2.0
lxml                               4.3.2
MarkupSafe                         1.1.1
matplotlib                         2.2.3
mccabe                             0.6.1
menuinst                           1.4.16
mistune                            0.8.4
mkl-fft                            1.0.10
more-itertools                     5.0.0
mpmath                             1.1.0
msgpack                            0.6.1
multipledispatch                   0.6.0
navigator-updater                  0.2.1
nbconvert                          5.4.1
nbformat                           4.4.0
networkx                           2.2
nltk                               3.4
nose                               1.3.7
notebook                           5.7.8
numba                              0.43.1+0.g8dabe7abe.dirty
numexpr                            2.6.9
numpy                              1.16.2
numpydoc                           0.8.0
olefile                            0.46
openpyxl                           2.6.1
packaging                          19.0
pandas                             0.24.2
pandocfilters                      1.4.2
parso                              0.3.4
partd                              0.3.10
path.py                            11.5.0
pathlib2                           2.3.3
patsy                              0.5.1
pep8                               1.7.1
pickleshare                        0.7.5
Pillow                             5.4.1
pip                                19.0.3
pkginfo                            1.5.0.1
pluggy                             0.9.0
ply                                3.11
prometheus-client                  0.6.0
prompt-toolkit                     1.0.15
psutil                             5.5.0
py                                 1.8.0
pycodestyle                        2.5.0
pycosat                            0.6.3
pycparser                          2.19
pycrypto                           2.6.1
pycurl                             7.43.0.2
pyflakes                           2.1.1
Pygments                           2.3.1
pylint                             1.9.2
pyodbc                             4.0.26
pyOpenSSL                          19.0.0
pyparsing                          2.3.1
pyreadline                         2.1
pyrsistent                         0.14.11
PySocks                            1.6.8
pytest                             4.3.1
python-dateutil                    2.8.0
pytz                               2018.9
PyWavelets                         1.0.2
pywin32                            223
pywinpty                           0.5.5
PyYAML                             5.1
pyzmq                              18.0.0
QtAwesome                          0.5.7
qtconsole                          4.4.3
QtPy                               1.7.0
requests                           2.21.0
rope                               0.12.0
ruamel-yaml                        0.15.46
scandir                            1.10.0
scikit-image                       0.14.2
scikit-learn                       0.20.3
scipy                              1.2.1
seaborn                            0.9.0
Send2Trash                         1.5.0
setuptools                         40.8.0
simplegeneric                      0.8.1
singledispatch                     3.4.0.3
six                                1.12.0
snowballstemmer                    1.2.1
sortedcollections                  1.1.2
sortedcontainers                   2.1.0
soupsieve                          1.8
Sphinx                             1.8.5
sphinxcontrib-websupport           1.1.0
spyder                             3.3.2
spyder-kernels                     0.3.0
SQLAlchemy                         1.3.1
statsmodels                        0.9.0
subprocess32                       3.5.3
sympy                              1.3
tables                             3.5.1
tblib                              1.3.2
terminado                          0.8.1
testpath                           0.4.2
toolz                              0.9.0
tornado                            5.1.1
tqdm                               4.31.1
traceback2                         1.4.0
traitlets                          4.3.2
typing                             3.6.6
unicodecsv                         0.14.1
unittest2                          1.1.0
urllib3                            1.24.1
wcwidth                            0.1.7
webencodings                       0.5.1
Werkzeug                           0.14.1
wheel                              0.33.1
widgetsnbextension                 3.4.2
win-inet-pton                      1.1.0
win-unicode-console                0.5
wincertstore                       0.2
wrapt                              1.11.1
xlrd                               1.2.0
XlsxWriter                         1.1.5
xlwings                            0.15.4
xlwt                               1.3.0
zict                               0.1.4
zipp                               0.3.3
![Issue_01](https://user-images.githubusercontent.com/20404987/61667987-51d9b100-ac90-11e9-9e94-f33304556b35.PNG)
![Issue_01](https://user-images.githubusercontent.com/20404987/61668002-5aca8280-ac90-11e9-8cdc-ce311a480530.PNG)

**Comment by honey09280** [comment]
Hello,

I just tried:

conda update setuptools


The new dependencies are: 

alabaster                          0.7.12
anaconda-client                    1.7.2
anaconda-navigator                 1.9.7
anaconda-project                   0.8.3
asn1crypto                         0.24.0
astroid                            1.6.5
astropy                            2.0.9
atomicwrites                       1.3.0
attrs                              19.1.0
Babel                              2.7.0
backports-abc                      0.5
backports.functools-lru-cache      1.5
backports.os                       0.1.1
backports.shutil-get-terminal-size 1.0.0
backports.shutil-which             3.5.2
backports.ssl-match-hostname       3.7.0.1
beautifulsoup4                     4.7.1
bitarray                           0.9.2
bkcharts                           0.2
bleach                             3.1.0
bokeh                              1.2.0
boto                               2.49.0
Bottleneck                         1.2.1
cdecimal                           2.3
certifi                            2019.6.16
cffi                               1.12.3
chardet                            3.0.4
Click                              7.0
cloudpickle                        1.2.1
clyent                             1.2.2
colorama                           0.4.1
comtypes                           1.1.7
conda                              4.7.10
conda-build                        3.17.8
conda-package-handling             1.3.11
conda-verify                       3.1.1
configparser                       3.7.4
contextlib2                        0.5.5
cryptography                       2.4.2
cycler                             0.10.0
Cython                             0.29.12
cytoolz                            0.10.0
dask                               1.2.2
decorator                          4.4.0
defusedxml                         0.6.0
distributed                        1.28.1
docutils                           0.14
entrypoints                        0.3
enum34                             1.1.6
et-xmlfile                         1.0.1
fastcache                          1.1.0
filelock                           3.0.12
Flask                              1.1.1
funcsigs                           1.0.2
functools32                        3.2.3.post2
future                             0.17.1
futures                            3.3.0
gevent                             1.4.0
glob2                              0.7
greenlet                           0.4.15
grin                               1.2.1
h5py                               2.9.0
heapdict                           1.0.0
html5lib                           1.0.1
idna                               2.8
imageio                            2.5.0
imagesize                          1.1.0
importlib-metadata                 0.17
ipaddress                          1.0.22
ipykernel                          4.10.0
ipython                            5.8.0
ipython-genutils                   0.2.0
ipywidgets                         7.5.0
isort                              4.3.21
itsdangerous                       1.1.0
jdcal                              1.4.1
jedi                               0.13.3
Jinja2                             2.10.1
jsonschema                         3.0.1
jupyter                            1.0.0
jupyter-client                     5.3.1
jupyter-console                    5.2.0
jupyter-core                       4.5.0
jupyterlab                         0.33.11
jupyterlab-launcher                0.11.2
keyring                            18.0.0
kiwisolver                         1.1.0
lazy-object-proxy                  1.4.1
libarchive-c                       2.8
linecache2                         1.0.0
llvmlite                           0.29.0
locket                             0.2.0
lxml                               4.3.4
MarkupSafe                         1.1.1
matplotlib                         2.2.3
mccabe                             0.6.1
menuinst                           1.4.16
mistune                            0.8.4
mkl-fft                            1.0.12
mkl-service                        2.0.2
mock                               3.0.5
more-itertools                     5.0.0
mpmath                             1.1.0
msgpack                            0.6.1
multipledispatch                   0.6.0
navigator-updater                  0.2.1
nbconvert                          5.5.0
nbformat                           4.4.0
networkx                           2.2
nltk                               3.4.4
nose                               1.3.7
notebook                           5.7.8
numba                              0.45.0+0.g7e2fa9823.dirty
numexpr                            2.6.9
numpy                              1.16.3
numpydoc                           0.9.1
olefile                            0.46
openpyxl                           2.6.2
packaging                          19.0
pandas                             0.24.2
pandocfilters                      1.4.2
parso                              0.5.0
partd                              1.0.0
path.py                            11.5.0
pathlib2                           2.3.4
patsy                              0.5.1
pep8                               1.7.1
pickleshare                        0.7.5
Pillow                             6.1.0
pip                                19.1.1
pkginfo                            1.5.0.1
pluggy                             0.12.0
ply                                3.11
prometheus-client                  0.7.1
prompt-toolkit                     1.0.15
psutil                             5.6.3
py                                 1.8.0
pycodestyle                        2.5.0
pycosat                            0.6.3
pycparser                          2.19
pycrypto                           2.6.1
pycurl                             7.43.0.2
pyflakes                           2.1.1
Pygments                           2.4.2
pylint                             1.9.2
pyodbc                             4.0.26
pyOpenSSL                          19.0.0
pyparsing                          2.4.0
pyreadline                         2.1
pyrsistent                         0.14.11
PySocks                            1.7.0
pytest                             4.6.2
python-dateutil                    2.8.0
pytz                               2019.1
PyWavelets                         1.0.3
pywin32                            223
pywinpty                           0.5.5
PyYAML                             5.1.1
pyzmq                              18.0.0
QtAwesome                          0.5.7
qtconsole                          4.5.1
QtPy                               1.8.0
requests                           2.22.0
rope                               0.14.0
ruamel-yaml                        0.15.46
scandir                            1.10.0
scikit-image                       0.14.2
scikit-learn                       0.20.3
scipy                              1.2.1
seaborn                            0.9.0
Send2Trash                         1.5.0
setuptools                         41.0.1
simplegeneric                      0.8.1
singledispatch                     3.4.0.3
six                                1.12.0
snowballstemmer                    1.9.0
sortedcollections                  1.1.2
sortedcontainers                   2.1.0
soupsieve                          1.8
Sphinx                             1.8.5
sphinxcontrib-websupport           1.1.2
spyder                             3.3.2
spyder-kernels                     0.3.0
SQLAlchemy                         1.3.5
statsmodels                        0.10.0
subprocess32                       3.5.4
sympy                              1.4
tables                             3.5.2
tblib                              1.4.0
terminado                          0.8.2
testpath                           0.4.2
toolz                              0.10.0
tornado                            5.1.1
tqdm                               4.32.1
traceback2                         1.4.0
traitlets                          4.3.2
typing                             3.7.4
unicodecsv                         0.14.1
unittest2                          1.1.0
urllib3                            1.24.2
wcwidth                            0.1.7
webencodings                       0.5.1
Werkzeug                           0.15.4
wheel                              0.33.4
widgetsnbextension                 3.5.0
win-inet-pton                      1.1.0
win-unicode-console                0.5
wincertstore                       0.2
wrapt                              1.11.2
xlrd                               1.2.0
XlsxWriter                         1.1.8
xlwings                            0.15.8
xlwt                               1.3.0
zict                               1.0.0
zipp                               0.5.1


But the error is still there. So kindly assist me accordingly.


![Issue_01](https://user-images.githubusercontent.com/20404987/61668709-64ed8080-ac92-11e9-8391-ea976eb37606.PNG)



Thanks,
Sai..

**Comment by ccordoba12** [comment]
Please run in a terminal (i.e. cmd.exe)

    spyder --reset

That should fix your problem.

**Comment by honey09280** [comment]
Hello @ccordoba12 sir,

I still have this issue, the issue was not resolved. The other day I made a mistake in copy pasting my dependencies (I copy pasted the Anaconda dependencies, not the Python dependencies). I apologize for my mistake. These are the dependencies I have at the moment (in my python).


alabaster                          0.7.12
astroid                            1.6.6
attrs                              19.1.0
Babel                              2.7.0
backports-abc                      0.5
backports.functools-lru-cache      1.5
backports.shutil-get-terminal-size 1.0.0
bleach                             3.1.0
certifi                            2019.6.16
chardet                            3.0.4
cloudpickle                        1.2.1
colorama                           0.4.1
configparser                       3.7.4
cycler                             0.10.0
decorator                          4.4.0
defusedxml                         0.6.0
docutils                           0.15
entrypoints                        0.3
enum34                             1.1.6
functools32                        3.2.3.post2
futures                            3.3.0
idna                               2.8
imagesize                          1.1.0
ipykernel                          4.10.0
ipython                            5.8.0
ipython-genutils                   0.2.0
isort                              4.3.21
jedi                               0.14.1
Jinja2                             2.10.1
jsonschema                         3.0.1
jupyter-client                     5.3.1
jupyter-core                       4.5.0
keyring                            18.0.1
kiwisolver                         1.1.0
lazy-object-proxy                  1.4.1
MarkupSafe                         1.1.1
matplotlib                         2.2.4
mccabe                             0.6.1
mistune                            0.8.4
nbconvert                          5.5.0
nbformat                           4.4.0
numpy                              1.16.4
numpydoc                           0.9.1
packaging                          19.0
pandocfilters                      1.4.2
parso                              0.5.1
pathlib2                           2.3.4
pickleshare                        0.7.5
Pillow                             6.0.0
pip                                19.1.1
prompt-toolkit                     1.0.16
psutil                             5.6.3
pycodestyle                        2.5.0
pyflakes                           2.1.1
Pygments                           2.4.2
pylint                             1.9.5
pyparsing                          2.4.0
pyrsistent                         0.15.3
python-dateutil                    2.8.0
pytz                               2019.1
pywin32-ctypes                     0.2.0
pyzmq                              18.0.2
QtAwesome                          0.5.8
qtconsole                          4.5.1
QtPy                               1.9.0
requests                           2.22.0
rope                               0.14.0
scandir                            1.10.0
setuptools                         41.0.1
simplegeneric                      0.8.1
singledispatch                     3.4.0.3
six                                1.12.0
snowballstemmer                    1.9.0
Sphinx                             1.8.5
sphinxcontrib-websupport           1.1.2
spyder                             3.3.6
spyder-kernels                     0.5.1
testpath                           0.4.2
tornado                            5.1.1
traitlets                          4.3.2
typing                             3.7.4
urllib3                            1.25.3
wcwidth                            0.1.7
webencodings                       0.5.1
wheel                              0.33.4
win-unicode-console                0.5
wrapt                              1.11.2


Previously I didn't have spyder in python. I installed it using pip install.


So I would request you to kindly help me ASAP @ccordoba12 sir.


Once again, I apologize for the inconvenience caused.


Thanks and Best,
Sai...
